src/System.Runtime/tests/ApiCompatBaseline.uapaot.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.MemoryExtensions.LastIndexOf(System.ReadOnlySpan<char>, System.ReadOnlySpan<char>, System.StringComparison)' does not exist in the implementation but it does exist in the contract.es not exist in the implementation but it does exist in the contract.
```

Comment:
Why do test projects require a baseline?

Reply:
This shouldn't be required anymore I believe. As we build System.Runtime for uapaot we needed that file to work around the build error when using a not yet ingested API.